src/org/labkey/test/components/ui/grids/ParentDetailPanelEdit.java

```diff
@@ -0,0 +1,164 @@
+package org.labkey.test.components.ui.grids;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.Component;
+import org.labkey.test.components.WebDriverComponent;
+import org.labkey.test.components.react.ReactSelect;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+/*
+    implements internal/components/ParentEntityEditPanel, in its edit mode.
+
+ */
+public class ParentDetailPanelEdit extends WebDriverComponent<ParentDetailPanelEdit.ElementCache>
```

Comment:
Note:
There already is a org.labkey.test.components.ui.entities.ParentEntityEditPanel I don't think this class is needed.

Reply:
Thanks, I didn't spot that.  Refactoring now to use it instead